mteb/tasks/PairClassification/por/Assin2RTE.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import annotations
+
+from mteb.abstasks.AbsTaskPairClassification import AbsTaskPairClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class Assin2RTE(AbsTaskPairClassification):
+    metadata = TaskMetadata(
+        name="assin2-rte",
```

Comment:
Maybe name the task as the class to follow the same convention
```suggestion
        name="Assin2RTE",
```